framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/Logo/Logo.module.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../styles/constants';
+
+.wrapper {
+	display: inline-flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	height: 53px;
+	color: $color-white;
+}
+
+.img {
+	width: auto;
+	height: 100%;
+}
+
+.branding {
+	margin: 14px 10px 1px 11px;
+	flex-direction: column;
+	display: inline-flex;
+	justify-content: space-between;
+}
+
+.brandingText {
+	text-transform: uppercase;
+	font-size: 14px;
+	font-weight: 600;
+	letter-spacing: 1.23px;
```

Comment:
line-height: 1.67 is missing. Maybe it's better to add this style as one of the type in `Text` component

Reply:
Yes I didn't added the line-height because I justified content with flex.